src/Storage/FileStorage.php

```diff
@@ -68,8 +68,9 @@ public function flush(): void
             $jsonData = $varDumper->asJson();
             $this->filesystem->write($basePath . 'data.json', $jsonData);
 
-            $jsonObjects = $varDumper->asJsonObjectsMap();
-            $this->filesystem->write($basePath . 'objects.json', $jsonObjects);
+            $jsonObjects = json_decode($varDumper->asJsonObjectsMap(), true);
```

Comment:
Forming a string then decoding then reindexing then encoding? That looks suboptimal... and `asJsonObjectsMap()` isn't used anywhere else so I'd make it a method that does everything w/o encoding/decoding/encoding.